index.html

```diff
@@ -238,6 +244,84 @@ <h2>About me</h2>
           </ul>
         </div>
       </section>
+      <!-- Contact section -->
+      <footer id="contact">
```

Comment:
- [ ] @letarulinda and @adel-gu kindly ensure to use flexbox to layout elements in your form, its a project requirement.

Reply:
We, Will, do so, thank you for the feedback.